notebooks/hybrid_rag_pipeline_with_breakpoints.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>Document Store Initialization</p><p>Let's create a simple document store with sample documents and their embeddings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>